lib/ex_doc/formatter/html/templates.ex

```diff
@@ -84,12 +84,32 @@ defmodule ExDoc.Formatter.HTML.Templates do
     for extra <- extras do
       %{id: id, title: title, group: group, content: content} = extra
 
+      search_data =
+        case extra do
```

Comment:
Move this clause perhaps to the conditional below? We don't need to have two conditionals :)